### PR TITLE
[Support] Small updates

### PR DESCRIPTION
**Pull request by alloy**
Some small additions that could help the next person.

### CODE REVIEW

File: playbooks/support/responding.md [hunk 1]
```diff
@@ -26,6 +26,9 @@ respond to these issues.
    - Take your best guess at the other information, it can always be edited later.
 
 2. Using the slack integration on the Jira Ops ticket, create a new slack channel to discuss resolution.
+
+   _Jira integrations are known to be flakey. If you have trouble getting the integration to work, simply create a channel named `#incident-N` (where `N` corresponds to the Jira Ops ticket) and link to it from the ticket._
```
Comment (sweir27) [inline playbooks/support/responding.md]: Oh gosh... this didn't work either? 😭

**Comment by sweir27** [comment]
💯 💯 Thank you for updating!!!

File: playbooks/support/responding.md (hunk 1 above)
Comment (alloy) [reply playbooks/support/responding.md]: It didn’t for @izakp, it did for me but that seems like the definition of flakey :)

Comment (izakp) [reply playbooks/support/responding.md]: It worked when I created Incident-31 but yes, the integration is definitely flakey

**Pull request merged**